torch/distributed/rpc/__init__.py

```diff
@@ -50,15 +49,24 @@ def init_rpc(
                         ``Worker1``) Name can only contain number, alphabet,
                         underscore, and/or dash, and must be shorter than
                         128 characters.
-            init_method(str): backend specific init arguments.
+            init_method(str): URL specifying how to initialize the
```

Comment:
Why do we still have this in the docs? Isn't this parameter removed from `init_rpc` now?